pkg/reconciler/pipelinerun/affinity_assistant_test.go

```diff
@@ -754,53 +871,6 @@ func TestThatCleanupIsAvoidedIfAssistantIsDisabled(t *testing.T) {
 	}
 }
 
-func TestDisableAffinityAssistant(t *testing.T) {
```

Comment:
why remove these tests?

Reply:
function `isAffinityAssistantDisabled` is now no longer used and removed

Reply:
sorry, missed this!